examples/end-to-end-session-based/02-End-to-end-session-based-with-Yoochoose-PyT.ipynb

```diff
@@ -112,10 +112,10 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    for col_name, col_schema in input_schema.column_schemas.items():
</pre><p>the for loop is not required and it is creating issue since it returns the input schema to non ragged. Please remove the for loop.</p><p><code></code><code></p><p>for col_name, col_schema in input_schema.column_schemas.items():</p><p>&nbsp;&nbsp;input_schema[col_name] = input_schema[col_name].with_shape((None, sparse_max[col_name]))</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/680/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
change made